src/main/scala/cognite/spark/v1/udf/CogniteUdfs.scala

```diff
@@ -0,0 +1,143 @@
+package cognite.spark.v1.udf
+
+import cats.effect.IO
+import cognite.spark.v1.Constants.{DefaultBaseUrl, DefaultMaxRetries}
+import cognite.spark.v1.{CdfSparkException, CdpConnector, Constants}
+import com.cognite.sdk.scala.common.ApiKeyAuth
+import com.cognite.sdk.scala.v1.{FunctionCall, GenericClient}
+import com.softwaremill.sttp.SttpBackend
+import io.circe.{Json, JsonObject, parser}
+import org.apache.spark.sql.SparkSession
+
+class CogniteUdfs(sparkSession: SparkSession) extends Serializable {
+
+  def initializeUdfs(
+      apiKey: ApiKeyAuth,
+      baseUrl: Option[String] = None,
+      maxRetries: Option[Int] = None): Unit = {
```

Comment:
Easier to just do `baseUrl: String = DefaultBaseUrl, maxRetries: Int = DefaultMaxRetries)`